Add slash command "/format" to automatically format PRs (#646)
GenericMappingTools/pygmt: CONTRIBUTING.md

make format
```

Don't worry if you forget to do it. Our continuous integration systems will
warn us and you can make a new commit with the formatted code.

> Even better, you can just write `/format` in the first line of any comments to
> automatically format your codes.

Reviewer: ```suggestion
Even better, you can just write `/format` in the first line of any comment in a
Pull Request to lint the code automatically.
```
